<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="options" native="true"/>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="createBtn">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyBtn">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetBtn">
          <property name="text">
           <string>Reset Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpBtn">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
